changelog-endpoint/src/main/kotlin/jp/co/soramitsu/bootstrap/changelog/parser/ChangelogParser.kt

```diff
@@ -4,6 +4,8 @@ import groovy.lang.GroovyClassLoader
 import jp.co.soramitsu.bootstrap.changelog.ChangelogInterface
 import org.springframework.stereotype.Component
 
+private val irohaKeyRegexp = Regex("[A-Za-z0-9_]{1,64}")
```

Comment:
`Regex("[A-Za-z0-9]{64}")` 🤔

Reply:
or what droes it stand for? I thought of iroha public key 😄

Reply:
It's Iroha `setAccountDetail` key regrex.